drivers/mspi/mspi_nrfe.c

```diff
@@ -519,8 +519,12 @@ static int xfer_packet(struct mspi_xfer_packet *packet, uint32_t timeout)
 
 	/* Wait for the transfer to complete and receive data. */
 	if ((packet->dir == MSPI_RX) && (ipc_receive_buffer != NULL) && (ipc_received > 0)) {
-		memcpy((void *)packet->data_buf, (void *)ipc_receive_buffer, ipc_received);
-		packet->num_bytes = ipc_received;
+		/*
+		 * Writing to packet->num_bytes causes memory access errors, so for now
+		 *	number of received data is stored in the first byte in data buffer.
```

Comment:
```suggestion
		 * number of received data is stored in the first byte in data buffer.
```
if you are use a gui for editing, the settings appear to be wrong